cfg/signal.js

```diff
@@ -1,5 +1,6 @@
 // PURPOSE: verify that report on signal can be configured
 // RUN: node --experimental-report signal.js
+// NOTICE: this doesn't yet work on windows
```

Comment:
can we say instead:
`Signal based report generation is not supported in Windows.` ? and probably quote the documentation - https://nodejs.org/dist/latest-v13.x/docs/api/report.html ? thanks.

(here as well as in other places down)

Reply:
updated